Makefile.PL

```diff
@@ -23,7 +23,7 @@ requires(
     'Net::Interface'     => 0,
     'MooseX::Getopt'     => 0,
     'Text::Reflow'       => 0,
-    'Zonemaster::Engine' => 4.0,
+    'Zonemaster::Engine' => 4.001,
```

Comment:
Does CLI make use of the new features introduced in Engine? Otherwise it's better not raise the version requirement.

Reply:
If someone updates CLI by "cpanm Zonemaster::CLI" we want that to pull updated Engine too.

I have followed the instruction "[Update Makefile.PL with required version]".

We have had this discussion before. We have tested CLI with the updated Engine, v4.1.0, not with v4.0.0.

[Update Makefile.PL with required version]: https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/ReleaseProcess-release.md#5-update-makefilepl-with-required-version